test/test_dbc.cpp

```diff
@@ -49,4 +49,80 @@ TEST_CASE("Testing dbc file loading", "[fileio]") {
 		REQUIRE(parser->get_messages().front().signals == msg.signals);
 	}
 
-}
\ No newline at end of file
+}
+
+/*!
+ * \brief TEST_CASE
+ * Test negative values in offset, min, max
+ */

```

Comment:
Suggest removing this since the test case name is stating this and I don't think I config doxygen to run on the test source files.

Reply:
changed in 6b4061c8759789bdc5138227d4f5fa564ef9304e